plugins/AudioFileProcessor/AudioFileProcessor.cpp

```diff
@@ -209,8 +210,9 @@ void AudioFileProcessor::deleteNotePluginData( NotePlayHandle * _n )
 
 void AudioFileProcessor::saveSettings(QDomDocument& doc, QDomElement& elem)
 {
-	elem.setAttribute("src", m_sample.sampleFile());
-	if (m_sample.sampleFile().isEmpty())
+	auto sampleFile = m_sample.sampleFileRelative();
```

Comment:
Given my suggestion of not coupling the `SampleBuffer` to the management of file paths and providing some abstraction layer that the `SampleBuffer` can use, this turns into:
```suggestion
	auto sampleFile = PathUtil::toShortestRelative(m_sample.sampleFile());
```

I expect something similar is happening in a variety of places, so this can be used as an example.